cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -122,9 +122,13 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementHelper {
       }
     }
 
+  private def findMember(cluster: Cluster, memberAddress: String): Option[Member] =
+    cluster.readView.members.find(
+        m ⇒ s"${m.uniqueAddress.address}" == memberAddress || m.uniqueAddress.address.hostPort == memberAddress)
```

Comment:
good. Could have crazy edge cases when people restart into using artery suddenly, but that's crazy and let's not care here...